lib/ClangImporter/ClangImporter.cpp

```diff
@@ -506,6 +506,24 @@ void importer::getNormalInvocationArguments(
       "-isystem", searchPathOpts.RuntimeResourcePath,
   });
 
+  if (ctx.LangOpts.isUsingCustomCxxStdLib()) {
+    // Add a define that ensures Clang differentiates between imported Clang
+    // modules with a
+    // custom libc++ path and regular Clang modules that use system's C++
+    // standard library.
+    invocationArgStrs.insert(invocationArgStrs.end(),
+                             {"-D__swift_use_custom_libcxx__"});
```

Comment:
Can we use `custom_stl` instead?

Reply:
switching to `__swift_use_custom_cxx_stlib__`